lib/flutter_pdfview.dart

```diff
@@ -70,14 +87,52 @@ class PDFView extends StatefulWidget {
   final bool enableSwipe;
   final bool swipeHorizontal;
   final String? password;
+
+  /// Works on [Android]
   final bool nightMode;
   final bool autoSpacing;
+
+  /// minimum swipe to change the page
   final bool pageFling;
+
+  /// Works on [Android]
+  ///
+  /// if [true] automatic center the page
+  ///
+  /// if [false] you can stop betwen two pages
   final bool pageSnap;
+
+  /// initial page to show
   final int defaultPage;
+
+  /// Works on [Android]
   final FitPolicy fitPolicy;
   final bool fitEachPage;
   final bool preventLinkNavigation;
+
+  /// Works on [Android]
+  ///
+  /// space betwen the pages (arround each page)
+  final int spacing;
+
+  /// Works on [Android]
+  ///
+  /// for zoom-in-out
+  final bool enableDoubleTap;
+
+  /// Works on [Android]
+  final Color setBackgroundColor;
+
+  /// Works on [Android]
+  final double setMaxZoom;
+
+  /// Works on [Android]
+  ///
+  /// for double tap (middle step)
+  final double setMidZoom;
+
+  /// Works on [Android]
```

Comment:
Plans to get this functional on iOS? 

I recently opened #210 due to unwanted default zoom of PDFView build, but am not experienced in Swift to develop a fix myself.